Assets/_Scripts/Button.cs

```diff
@@ -5,22 +5,22 @@
 
 public class Button : MonoBehaviour
 {
-    public Defender defenderPrefab;
-    private Button[] _buttonArray;
+    private Defender _defenderPrefab;
```

Comment:
A button class does not need a defender prefab, the name does not suggest that.

I recommend using something like `DefenderPlacerButton`